### ISSUE TITLE
Wrong logprobability calculation in `brax` networks

### ISSUE THREAD

**Issue by joeryjoery** [joeryjoery/trtpi#2]
See: https://github.com/joeryjoery/trtpi/blob/fa007f0136ffd34711c8eeda5d83cbcfd7cea718/experiment/models/networks/brax.py#L57-L67

I forgot to sum `logdet` across the dimensions. This should be:
```python
elif self.option == 'tanh': 
  
     # Change of variable rule; see Appendix C of 
     # https://arxiv.org/abs/1801.01290 
     u = jnp.arctanh(jnp.clip(action, -1 + 1e-6, 1 - 1e-6)) 
  
     lp = jax.scipy.stats.norm.logpdf(u, mean, scale)
  
     # Numerically stable log-determinant of tanh jacobian 
     logdet = 2 * (jnp.log(2) - u - jax.nn.softplus(-2 * u)) 
     return (lp - logdet).sum()
```

The consequence is that the learners now incorrectly expands these dimensions. Consquences for the experiments are:
- SMC + MCTS: The expanded dimensions get averaged in the loss (should be summed instead) making the policy coefficient artifically smaller. This is not intended, but shouldn't break the algorithm.
- PPO: The ratio-clipping now acts separately on each dimension for the tanh-correction only. This is wrong.
